packages/cli/src/environments.ee/source-control/source-control-import.service.ee.ts

```diff
@@ -109,9 +256,15 @@ export class SourceControlImportService {
 		);
 	}
 
-	async getLocalVersionIdsFromDb(): Promise<SourceControlWorkflowVersionId[]> {
+	async getAllLocalVersionIdsFromDb(): Promise<SourceControlWorkflowVersionId[]> {
 		const localWorkflows = await this.workflowRepository.find({
-			relations: ['parentFolder'],
+			relations: {
+				parentFolder: {
+					homeProject: {
```

Comment:
I don't see where this is used later on in the method. or is it just to prevent using string 'parentFolder' ?